LICENSE

```diff
@@ -211,3 +211,96 @@ Home page: https://github.com/gradle/gradle
 License: https://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
+
+This product includes code from Apache Iceberg.
+
+* spec/rest-catalog-open-api.yaml
+* integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisRestCatalogIntegrationTest.java
+
+Copyright: Copyright 2017-2025 The Apache Software Foundation
+Home page: https://iceberg.apache.org
+License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes code from OpenAPITool openapi-generator
+
+* server-templates/formParams.mustache
+* server-templates/apiService.mustache
+* server-templates/bodyParams.mustache
+* server-templates/pojo.mustache
+* server-templates/headerParams.mustache
+* server-templates/apiServiceImpl.mustache
+* server-templates/queryParams.mustache
+* server-templates/api.mustache
+
+Copyright: Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech) Copyright 2018 SmartBear Software
+Home page: https://openapi-generator.tech/
+License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes code from Google Docsy.
+
+* site/layouts/community/list.html
+* site/layouts/docs/baseof.html
+* site/layouts/partials/community_links.html
+* site/layouts/partials/head.html
+* site/layouts/partials/navbar.html
+* site/layouts/shortcodes/redoc-polaris.html
+
+Home page: https://www.docsy.dev/
+License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes code from Project Nessie.
```

Comment:
Were these not contributed to the project by Dremio employees? If that's the case, then I think the regular contribution rules would apply, even if the code is present in another project. It doesn't matter that it was originally in another project, it matters that it was directly contributed by the copyright owner. If the copyright owner was not the one to contribute, then the third-party documentation would be needed.

Reply:
But, I don't know if we are talking about the same original authors. @snazy can you confirm that you are the author of the original files ? or do we have other dremio contributors on these files ?

Reply:
There are multiple authors on this code

Reply:
@snazy thanks for the update, it helps.

@rdblue imho, we are in the same situate as for Gradle resources (e.g. `gradlew`, ...): as the code is copied from another project, this project is under the ALv2, so we should document where the code is coming and the copyright.

Reply:
Side note: `gradle/wrapper/gradle-wrapper.properties` is a _generated_ file, not "copied" from anywhere.

Reply:
@rdblue thoughts ?

Reply:
I guess it's fine to treat it as a third-party contribution if the attribution is unknown from the Nessie project.

I'm actually more concerned right now about the GPL reference that popped up in the Iceberg NOTICE changes.

Reply:
I checked and it's dual license: CDDL + GPL or GPL + CE.
It would be great if Nessie clarify in the NOTICE, but I don't think it's a blocker.

@rdblue thoughts ?

Reply:
I've updated Nessie `NOTICE` with the 0.102.5 version.